src/main/java/com/dataloom/neuron/Signal.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2017. Kryptnostic, Inc (dba Loom)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@thedataloom.com
+ */
+
+package com.dataloom.neuron;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.dataloom.client.serialization.SerializationConstants;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class Signal {
```

Comment:
In short, yes.

Since we're doing the neuron theme, a "signal" is just an event. `Signal` is meant to be the base class for all events, similar to `AbstractSecurableObject`, and `AuditableSignal` is meant to be the re-implementation of `AuditableEvent` to match the new `AUDIT_LOG` table.